apps/web/languages/pt.json

```diff
@@ -18,6 +18,11 @@
         "title": "Tenha seu próprio <0>espaço pessoal</0> e use-o como quiser.",
         "subtitle": "Crie sua conta para salvar e armazenar tudo que você fizer.",
         "aboutTheSiteButton": "Sobre esse site"
+      },
+      "utility": {
+        "signInButton": "Entrar",
+        "signUpButton": "Inscrever-se",
+        "popoverTitle": "Configurações"
```

Comment:
@MFelipeSilva I think is better to create another object inside utility object, ex. `"popover": { "title": "Settings" }`, what u think?

Reply:
corrected